.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
According to @ericcornelissen this file is in the .gitignore so you should delete this manually